package.json

```diff
@@ -10,7 +10,7 @@
     "react-linechart": "1.1.9",
     "react-redux": "5.0.5",
     "redux": "3.7.2",
-    "redux-mock-store": "1.2.3",
+    "redux-promise-middleware": "^4.3.0",
```

Comment:
@SSabu please remove the `^` here to lock the version into place. I notice some other dependencies that also need this to be done. Thanks!